x/gitopia/keeper/genesis.go

```diff
@@ -151,9 +152,17 @@ func InitGenesis(ctx sdk.Context, k Keeper, genState types.GenesisState) {
 	}
 
 	if !k.accountKeeper.HasAccount(ctx, k.accountKeeper.GetModuleAddress(types.LiquidityBootstrappingPoolAccountName)) {
-		totalLiquidityBootstrappingPoolCoins := sdk.NewCoin(params.BaseCoinUnit, sdk.NewInt(LIQUIDITY_BOOTSTRAPPING_POOL_AMOUNT))
+		coin := sdk.NewCoin(params.BaseCoinUnit, sdk.NewInt(LIQUIDITY_BOOTSTRAPPING_POOL_AMOUNT))
 
-		if err := k.createModuleAccount(ctx, types.LiquidityBootstrappingPoolAccountName, totalLiquidityBootstrappingPoolCoins); err != nil {
+		if err := k.createModuleAccount(ctx, types.LiquidityBootstrappingPoolAccountName, coin); err != nil {
+			panic(err)
+		}
+
+		addr := k.accountKeeper.GetModuleAddress(types.LiquidityBootstrappingPoolAccountName)
+		balances := k.bankKeeper.GetAllBalances(ctx, addr)
+
+		err := k.distrKeeper.FundCommunityPool(ctx, balances, addr)
```

Comment:
why do we move liquidity pool to community pool?

Reply:
@ProgramCpp We wanted the ability to move lbp funds via governance. Moving it into community pool enables us to do it via community spend proposal